eloop.c

```diff
@@ -181,22 +174,19 @@ static void eloop_sock_table_remove_sock(struct eloop_sock_table *table,
 
 
 static void eloop_sock_table_set_fds(struct eloop_sock_table *table,
-				     fd_set *fds)
+				     struct pollfd *fds)
 {
 	int i;
-
-	FD_ZERO(fds);
-
```

Comment:
There are a number of places where whitespace or blank lines has been stripped out. I would caution you to be careful to try and preserve the original code formatting when that is not the primary purpose for the code change. In particular, I generall prefer a blank line after variable declarations before the first statement of the block.